node-api/src/storage.rs

```diff
@@ -12,8 +12,6 @@
 */
 
 //! For querying runtime storage.
-//!
```

Comment:
Removed that comment because redundant. The files, copied or not, indicate this via the header already.